docs/credentials.rst

```diff
@@ -0,0 +1,389 @@
+
+.. _ug_credentials_ocp_k8s:
+
+OpenShift or Kubernetes API Bearer Token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index::
+   pair: credential types; OpenShift
+   pair: credential types; Kubernetes
+   pair: credential types; API bearer token
+
+
+Selecting this credential type allows you to create instance groups that point to a Kubernetes or OpenShift container.
+
+|Credentials - create Containers credential|
+
+.. |Credentials - create Containers credential| image:: _static/images/credentials-create-containers-credential.png
+    :alt: Credentials - create Containers credential form
+
+Container credentials have the following inputs:
+
+- **OpenShift or Kubernetes API Endpoint** (required): the endpoint to be used to connect to an OpenShift or Kubernetes container
+- **API Authentication Bearer Token** (required): The token to use to authenticate the connection
+- **Verify SSL**: Optionally you can check this option to verify the server’s SSL certificate is valid and trusted. Environments that use internal or private CA’s should leave this option unchecked to disable verification.
+- **Certificate Authority Data**: include the ``BEGIN CERTIFICATE`` and ``END CERTIFICATE`` lines when pasting the certificate, if provided
+
+
+A ``ContainerGroup`` is a type of ``InstanceGroup`` that has an associated Credential that allows for connecting to an OpenShift cluster. To set up a container group, you must first have the following:
+
+- A namespace you can launch into (every cluster has a “default” namespace, but you may want to use a specific namespace)
+- A service account that has the roles that allow it to launch and manage Pods in this namespace
+- If you will be using execution environments in a private registry, and have a Container Registry credential associated to them in AWX, the service account also needs the roles to get, create, and delete secrets in the namespace. If you do not want to give these roles to the service account, you can pre-create the ``ImagePullSecrets`` and specify them on the pod spec for the ContainerGroup. In this case, the execution environment should NOT have a Container Registry credential associated, or AWX will attempt to create the secret for you in the namespace.
+- A token associated with that service account (OpenShift or Kubernetes Bearer Token)
+- A CA certificate associated with the cluster
+
+Create a Service Account in an Openshift cluster (or K8s) in order to be used to run jobs in a container group via AWX. After the Service Account is created, its credentials are provided to AWX in the form of an Openshift or Kubernetes API bearer token credential. Below describes how to create a service account and collect the needed information for configuring AWX.
+
+To configure AWX:
+
+1. To create a service account, you may download and use this sample service account, :download:`containergroup sa <_static/images/containergroup-sa.yml>` and modify it as needed to obtain the above credentials.
```

Comment:
Oh, is this where it's used? Create some other folder in the static dir, then. Maybe name it “samples” or something.